sympy/solvers/polysys.py

```diff
@@ -20,6 +20,24 @@ def solve_poly_system(seq, *gens, **args):
     """
     Solve a system of polynomial equations.
 
+    Parameters
+    ==========
+
+    seq: a list/tuple/set
+        Listing all the equations that are needed to be solved
+    *gens: symbols
```

Comment:
I think `gens` is more general than symbols, e.g. maybe it can be any generator of the poly like `cos(x)` in `Poly(cos(x)-1)`.

Reply:
@smichr Sure I will update it.